### PR TITLE
docs: add getting started guide

### PR DESCRIPTION
**Pull request by aaronsteers**
----
:books: Documentation preview :books:: https://meltano-edk--33.org.readthedocs.build/en/33/

### CODE REVIEW

**Comment by aaronsteers** [comment]
pre-commit fix

**Comment by aaronsteers** [comment]
pre-commit.ci fix

**Comment by aaronsteers** [comment]
pre-commit.ci autofix

**Review by tayloramurphy — APPROVED** [review]

File: docs/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Getting Started with the EDK
+
+The Meltano extension developer kit is the fastest way to build custom Meltano extensions. Extensions allow you to integrate any existing data tool into your existing Meltano projects - or build your own.
```
Comment (tayloramurphy) [inline docs/getting_started.md:3]:
```suggestion
The Meltano Extension Developer Kit (EDK) is the fastest way to build custom Meltano extensions. Extensions allow you to integrate any existing data tool into your existing Meltano projects - or build your own.
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## What can I build with the EDK?
+
+For examples of types of tools you can build or wrap with the EDK, see the MeltanoHub [Utilities](https://hub.meltano.com/utilities/), [Orchestrators](https://hub.meltano.com/orchestrators), and [Transformers](https://hub.meltano.com/transformers/) lists.
```
Comment (tayloramurphy) [inline docs/getting_started.md:7]:
```suggestion
For examples of types of tools you can build or wrap with the EDK, see the list of [EDK Preview Utilities](https://hub.meltano.com/utilities/) on Meltano Hub.
```

**Pull request merged**